ui/components/__tests__/AlertsTable.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { makeEventSourceLink } from "../AlertsTable";
+
+describe("AlertsTable", () => {
+  describe("makeEventSourceLink", () => {
+    it("should convert CrossNamespaceRef objects to urls with filters", () => {
+      const allNames = {
+        apiVersion: "v1",
+        name: "*",
+        namespace: "space",
+        kind: "GitRepository",
+        matchLabels: [],
+      };
+      const sourceLink = makeEventSourceLink(allNames);
+      expect(sourceLink.includes("/sources")).toEqual(true);
+      expect(
+        sourceLink.includes("type") && sourceLink.includes("GitRepository")
+      ).toEqual(true);
+      console.log(sourceLink);
```

Comment:
Great tests!

The only question: does this call to `console.log` have any special purpose?

Reply:
UGH. I'm fired. Thanks for finding that

Reply:
*side-whispering* Great that I cover my tracks well. Since _some_ totally random time I always search in files for `console.log` and `fmt.Println` before submitting my PRs for review. 🤣